src/test/index.js

```diff
@@ -18,8 +18,7 @@ describe( 'Test Jetpack blocks', () => {
 			() => jest.fn()
 		);
 
-		const setupJetpackEditor = require( '../jetpack-editor-setup' ).default;
-		setupJetpackEditor( { blogId: 1, isJetpackActive: true } );
+		require( '../allowed-blocks-setup' );
```

Comment:
I don't personally prefer to use inline `require`s. The test style was already written as such. I just knew that test was going to fail because of my deletion of that module. My "fixing" that test from failing was neither an indication that I considered this PR tested with respect to the dis/allow lists, nor an indication of my preference in importing modules.

That being said, the unfortunate part of that test is that there wouldn't be a simple way to move it from being an inline `require` to an `import` operator at the top of the test module because the module being required is the subject under test and the test is asserting that the `jetpack` block collection is implicitly being registered when `import './shared/block-category';` happens from `jetpack/projects/plugins/jetpack/extensions/editor.native.js`.

I can still adjust the implementation details of the test, but I'd have to give some thought to what that may even look like as I'm unsure of the practicality of the given test at the moment.

WDYT?

Reply:
Oh I see, ok, let's keep it this way for now. However, I'd consider adding a comment in the code explaining the reason for using the inline require, this way when we tackle this in the future we'll have a better context, wdyt?

Reply:
- [ ] Consider adding a comment in the code explaining the reason for using the inline require, this way when we tackle this in the future we'll have a better context.

Sounds good to me.

Reply:
I haven't started refining the tests yet. However, I'm consolidating remaining checklist items in the PR description. So I'm resolving this conversation in the meantime. I hope that is okay with you, @fluiddot . It's becoming difficult to manage a lot of threads.